sources/core/Stride.Core/Stride.Core.csproj

```diff
@@ -19,6 +19,14 @@
     <ExtrasUwpMetaPackageVersion>6.2.12</ExtrasUwpMetaPackageVersion>
   </PropertyGroup>
   
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net7.0|AnyCPU'">
+    <WarningLevel>7</WarningLevel>
```

Comment:
Can you explain this change and why it's conditional?

Reply:
I Must have been messing with too much when testing I have removed this as its not needed.